src/ngircd/channel.c

```diff
@@ -98,7 +98,6 @@ Channel_InitPredefined( void )
 	REQUEST Req;
 	const struct Conf_Channel *conf_chan;
 	char *c;
-	char modes[COMMAND_LEN], name[CHANNEL_NAME_LEN];
```

Comment:
Nope. Very old compilers don't support scoped variables at all, so (sadly) wen can't do that.